segmentation/utils/synthetic_spatial_datagen_test.py

```diff
@@ -102,3 +106,70 @@ def test_generate_test_label_map():
     # needing to index arrays, we couldn't 0-index the label values in label_map
     # because values of 0 in a label map are ignored by regionprops
     assert (np.sort(label_map_flat) == np.sort(centroid_indices) + 1).all()
+
+
+def test_generate_two_cell_test_segmentation_mask():
+    # this function tests the functionality of generating the segmentation mask
+
+    sample_segmentation_mask, sample_cell_centers = synthetic_spatial_datagen.generate_two_cell_test_segmentation_mask()
+
+    # assert that our labels are just blank, 1, and 2
+    assert set(np.unique(sample_segmentation_mask)) == set([0, 1, 2])
+
+    # assert that our centers are being labeled correctly
+    assert sample_segmentation_mask[sample_cell_centers[1][0], sample_cell_centers[1][0]] == 1
+    assert sample_segmentation_mask[sample_cell_centers[2][0], sample_cell_centers[2][1]] == 2
+
+    # using the default cell radius of 10, assert that the cells are next to each other
+    # we only include the offset columnwise because that's how the cells are generated next to each other
+    assert sample_segmentation_mask[sample_cell_centers[1][0], sample_cell_centers[1][1] + 10] == 2
+    assert sample_segmentation_mask[sample_cell_centers[2][0], sample_cell_centers[2][1] - 10] == 1
```

Comment:
just setting the cell radius to 10 in the function call (line 114) will guarantee that this test function doesn't break here (in case someone decides to change the default cell radius later).